### PR TITLE
Improve documentation for Android on Windows

### PR DESCRIPTION
**Pull request by nikita-bykov**
# Description
Improve documentation for Android on Windows to present it in more convenient way.
The result: 
image

#### Related issue:
[Improve documentation for Android tools on Windows and MacOS](https://github.com/actions/virtual-environments-internal/issues/1073)
## Check list
- [x] Related issue / work item is attached
- [x] Changes are tested and related VM images are successfully generated: [Windows 2019](https://github.visualstudio.com/virtual-environments/_build/results?buildId=81129&view=logs&j=011e1ec8-6569-5e69-4f06-baf193d1351e&t=486c0909-f67d-5748-80b3-bb1d531e7b43), [Windows 2016](https://github.visualstudio.com/virtual-environments/_build/results?buildId=81130&view=logs&j=011e1ec8-6569-5e69-4f06-baf193d1351e)

### CODE REVIEW

**Review by AlenaSviridenko — APPROVED** [review]

File: images/win/scripts/SoftwareReport/SoftwareReport.Android.psm1 [hunk 1]
```diff
@@ -1,27 +1,24 @@
...
-        [string] $Row
+        [string] $ComponentName
     )
-    return $Row.Split("|").Trim()
+    $path = Join-Path $env:ANDROID_HOME $ComponentName
+    return "Location $path"
 }
 
-function Create-AndroidTableObject {
+function Get-AndroidSDKRoot {
```
Comment (al-cheb) [inline images/win/scripts/SoftwareReport/SoftwareReport.Android.psm1:16]: Looks like the same as Get-AndroidComponentLocation

Comment (nikita-bykov) [reply images/win/scripts/SoftwareReport/SoftwareReport.Android.psm1:8]: Removed it

File: images/win/scripts/SoftwareReport/SoftwareReport.Android.psm1 [hunk 2]
```diff
@@ -42,68 +39,120 @@ function Get-AndroidInstalledPackages {
...
-    return $packageInfo | ForEach-Object {
-        $packageInfoParts = Split-AndroidSDKOutputRow $_
-        $packageName = $packageInfoParts[0].split(";")[1]
-        $packageDescription = $packageInfoParts[2] + ", Revision " + $packageInfoParts[1]
-        return Create-AndroidTableObject -PackageName $packageName -Description $packageDescription
+    $versions = $packageInfo | Where-Object { $_ -Match "Android SDK Platform " } | ForEach-Object {
+        $packageInfoParts = Split-TableRowByColumns $_
+        $revision = $packageInfoParts[1]
+        $version = $packageInfoParts[0].split(";")[1]
+        return "$version, (rev $revision)"
```
Comment (al-cheb) [inline images/win/scripts/SoftwareReport/SoftwareReport.Android.psm1:115]: remove `,`

Comment (nikita-bykov) [reply images/win/scripts/SoftwareReport/SoftwareReport.Android.psm1]: Removed it

**Review by maxim-lobanov — APPROVED** [review]

**Pull request merged**